src/modules/HomeModule.tsx

```diff
@@ -1,15 +1,20 @@
+import { useCallback } from 'react'
 import { Calendar } from 'react-feather'
 import { HomeDayCard } from '../components/molecules'
 import { HomePageProps } from '../types'
 
 const HomeModule: React.FC<HomePageProps> = ({ yesterday, today, tomorrow }) => {
+  const getCurrentDate = useCallback(() => {
```

Comment:
useMemo na to muzes pouzit, ne usecallback. A pls zkus tam dat opravdu dnesni datum a zkus to naformatovat tak, ze tam bude 17th November :D